in_text/text_04_predictions.yml

```diff
@@ -76,800 +76,1124 @@ cross-cites:
+      data-units: NA
     -
-      attr-label: TCDC
+      attr-label: issue_time
       attr-def: >-
-        Total cloud cover
-      attr-defs: https://www.nco.ncep.noaa.gov/pmb/products/gens/gec00.t00z.pgrb2s.0p25.f003.shtml
-      data-min: NA
-      data-max: NA
-      data-units: decimal fraction
+        forecast issue time
+      attr-defs: 
+      data-min: 2022-XX-XX # will complete when we have latest vbersion of this dataset
+      data-max: 2022-XX-XX # will complete when we have latest vbersion of this dataset
+      data-units: days since 
     -
-      attr-label: TMP
+      attr-label: time
       attr-def: >-
-        Air temperature
-      attr-defs: https://www.nco.ncep.noaa.gov/pmb/products/gens/gec00.t00z.pgrb2s.0p25.f003.shtml
-      data-min: NA
-      data-max: NA
-      data-units: degrees C
+        Forecast date
+      attr-defs: This data release
+      data-min: XX # will complete when we have latest vbersion of this dataset
+      data-max: XX # will complete when we have latest vbersion of this dataset
+      data-units: degrees Celsius
     -
-      attr-label: DSWRF
-      attr-def: >-
-        Downward short-wave radiation flux
-      attr-defs: https://www.nco.ncep.noaa.gov/pmb/products/gens/gec00.t00z.pgrb2s.0p25.f003.shtml
+      attr-label: model_name
+      attr-def:  >-
+        Model name. DA, no DA and persistence
+      attr-defs: This data release
       data-min: NA
       data-max: NA
-      data-units: Watts per meter squared
+      data-units: NA
     -
-      attr-label: WS
-      attr-def: >-
-        Wind speed
-      attr-defs: https://www.nco.ncep.noaa.gov/pmb/products/gens/gec00.t00z.pgrb2s.0p25.f003.shtml
-      data-min: NA
-      data-max: NA
-      data-units: meters per second
+      attr-label: max_temp_c_predicted
+      attr-def:  >-
+        Maximum temperature predicted in degrees C
+      attr-defs: This data release
+      data-min: 6.4 # will update when we have latest vbersion of this dataset
+      data-max: 24.6 # will update when we have latest vbersion of this dataset
+      data-units: degree C
     -
-      attr-label: TMAX
-      attr-def: >-
-        Maximum air temperature
-      attr-defs: https://www.nco.ncep.noaa.gov/pmb/products/gens/gec00.t00z.pgrb2s.0p25.f003.shtml
-      data-min: NA
-      data-max: NA
-      data-units: degrees C
+      attr-label: max_temp_l90
+      attr-def:  >-
+        XX
+      attr-defs: This data release 
+      data-min: 5.86 # will update when we have latest vbersion of this dataset
+      data-max: 22.49 # will update when we have latest vbersion of this dataset
+      data-units: degree C
     -
-      attr-label: TMIN
-      attr-def: >-
-        Minimum air temperature
-      attr-defs: https://www.nco.ncep.noaa.gov/pmb/products/gens/gec00.t00z.pgrb2s.0p25.f003.shtml
-      data-min: NA
-      data-max: NA
-      data-units: degrees C
+      attr-label: max_temp_u90
+      attr-def:  >-
+        XX
```

Comment:
Not sure how to describe this here, @jzwart @limnoliver @jdiaz4302. What is u90? do you know how we should describe this and can you provide suggestion?

Reply:
This is the upper end of the 90% Confidence interval for all maximum stream temperature predictions on a given date and segment.